### PR TITLE
WT3 backwash solids handling

### PR DESCRIPTION
**Pull request by adam-a-a**
## Summary/Motivation:
Backwash solids handling unit for Santa Barbara case study.

## Changes proposed in this PR:
- add unit model and tests

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by bknueven — DISMISSED** [review]

**Review by bknueven — DISMISSED** [review]

File: watertap/unit_models/zero_order/backwash_solids_handling_zo.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+from watertap.core import build_sido, pump_electricity, ZeroOrderBaseData
+
+# Some more information about this module
+__author__ = "Adam Atia"
+
+
+@declare_process_block_class("BackwashSolidsHandlingZO")
+class BackwashSolidsHandlingZOData(ZeroOrderBaseData):
+    """
+    Zero-Order model for a Dual Media Filtration unit operation.
```
Comment (kurbansitterley) [inline watertap/unit_models/zero_order/backwash_solids_handling_zo.py]: docstring should have Backwash Solids Handling

**Review by kurbansitterley — CHANGES_REQUESTED** [review]
An extra "operation." in the second line of the module docstring and wrong unit referenced in the Class docstring. Might as well address these now rather than later.

File: watertap/unit_models/zero_order/backwash_solids_handling_zo.py (hunk 1 above)
Comment (bknueven) [inline watertap/unit_models/zero_order/backwash_solids_handling_zo.py]:
```suggestion
    Zero-Order model for a Backwash Solids Handling unit operation.
```

File: watertap/unit_models/zero_order/backwash_solids_handling_zo.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+This module contains a zero-order representation of a backwash solids handling unit.
+operation.
```
Comment (bknueven) [inline watertap/unit_models/zero_order/backwash_solids_handling_zo.py]:
```suggestion
```

**Review by bknueven — APPROVED** [review]

**Review by kurbansitterley — APPROVED** [review]
LGTM

**Review by luohezhiming — APPROVED** [review]

**Pull request merged**